ara_plumes/tests/test_all.py

```diff
@@ -160,22 +155,55 @@ def test_convert_video_to_numpy_array():
     np.testing.assert_equal(result[0].shape, (100, 100))
 
 
-def test_circle_poly_intersection_real():
-    x0 = np.sqrt(1 / 2 * (-1 + np.sqrt(5)))
-    y0 = x0**2
-    expected = np.array([[-x0, y0], [x0, y0]])
-    result = utils.circle_poly_intersection(1, 0, 0, 1, 0, 0, True)
-    np.testing.assert_array_almost_equal(result, expected)
+def test_circle_poly_intersection():
```

Comment:
Where is `circle_poly_intersection()` used?

Reply:
it's a useful function for mapping the *flattened* points back onto the frames.